### PR TITLE
Uploaded FDRC-FLYDRAGONF722_V2.2.config and deleted the previous M file.

### PR DESCRIPTION
**Pull request by ZyonMo**
Uploaded FDRC-FLYDRAGONF722_V2.2.config and deleted the previous M file.

### CODE REVIEW

File: configs/FDRC-FLYDRAGONF722_V2.2.config [hunk 1]
```diff
@@ -1,17 +1,17 @@
-# Rotorflight / STM32F7X2 (S7X2) 4.3.0 Jan  9 2024 / 11:09:54 (norevision) MSP API: 12.3
+# Rotorflight / STM32F7X2 (S7X2) 4.3.0 May 15 2024 / 09:06:57 (05570fe) MSP API: 12.6
 
-board_name FLYDRAGONF722M_V2
+board_name FLYDRAGONF722_V2.2
```
Comment (rotorflight) [inline configs/FDRC-FLYDRAGONF722_V2.2.config]: Sorry but you can't use a dot in the board name. (nor dash).
Please replace the dot with an underscore, or use another name altogether.

**Review by ghost — CHANGES_REQUESTED** [review]
Don't use a dot in the board name

File: configs/FDRC-FLYDRAGONF722_V2.2.config (hunk 1 above)
Comment (ZyonMo) [reply configs/FDRC-FLYDRAGONF722_V2.2.config]: Oh, sorry, I'll change that.

File: configs/FDRC-FLYDRAGONF722_V2_2.config [hunk 2]
```diff
@@ -1,17 +1,17 @@
-# Rotorflight / STM32F7X2 (S7X2) 4.3.0 Jan  9 2024 / 11:09:54 (norevision) MSP API: 12.3
+# Rotorflight / STM32F7X2 (S7X2) 4.3.0 May 15 2024 / 09:06:57 (05570fe) MSP API: 12.6
 
-board_name FLYDRAGONF722M_V2
+board_name FLYDRAGONF722_V2.2
```
Comment (rotorflight) [inline configs/FDRC-FLYDRAGONF722_V2_2.config]: This must match the file name.

**Review by ghost — CHANGES_REQUESTED** [review]

File: configs/FDRC-FLYDRAGONF722_V2_2.config (hunk 2 above)
Comment (ZyonMo) [reply configs/FDRC-FLYDRAGONF722_V2_2.config]: Okay, the problem has been modified.

**Review by ghost — APPROVED** [review]

**Pull request merged**